soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -10,17 +10,6 @@
 
 #define EC_REG_BASE_ADDR 0x00f00000
 
-#ifndef FALSE
-#define FALSE   0
-#endif
-
-/* TRUE can be defined as !FALSE but defining
- * it as 1 allows it to fit into a bitfield.
- */
-#ifndef TRUE
-#define TRUE    1
-#endif
```

Comment:
Please put this change in a separate commit.

Reply:
@stephanosio 

Got it, sir.
I've put them in separate commits this time.
Thanks again.